apps/fluent-tester/src/TestComponents/Shadow/ShadowWithDifferentPropsTestSection.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Text } from '@fluentui/react-native';
+import { Shadow } from '@fluentui-react-native/experimental-shadow';
+import { Theme, useFluentTheme } from '@fluentui-react-native/framework';
+import { View } from 'react-native';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
+import { mergeStyles } from '@fluentui-react-native/merge-props';
+
+const getThemedStyles = themedStyleSheet((t: Theme) => {
+  return {
+    defaultShadowTestBoxProps: {
+      maxWidth: 732,
+      minHeight: 64,
+      padding: 20,
+      marginVertical: 16,
+      marginHorizontal: 32,
+      backgroundColor: t.colors.background,
+    },
+  };
+});
+
+export const ShadowWithDifferentPropsTestSection: React.FunctionComponent = () => {
```

Comment:
So that this isn't a "catch-all" test section, could we actually have different sections with different sets of props? Not sure if that makes sense / is over-engineering it.

Reply:
I thought about that, but I think each section would probably just have one test - feel like it makes more sense if different sections would have more than one test. But don't have a strong preference, I see the tester sometimes has sections that only have a test